components/Buttons/PrependIconButton.vue

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <BaseButton
+        v-bind="$attrs">
+        <template #prepend="{ color }">
+            <div :class="prependClasses">
+                <slot
+                    name="prepend"
+                    :color="color" />
+            </div>
+        </template>
+        <slot />
```

Comment:
It could be better to keep one slot names prepend and add everything to it - easier managing

Reply:
I use this slot for multi-button list.